src/mbgl/gl/context.cpp

```diff
@@ -164,15 +164,20 @@ void Context::verifyProgramLinkage(ProgramID program_) {
     throw std::runtime_error("program failed to link");
 }
 
-UniqueBuffer Context::createVertexBuffer(const void* data, std::size_t size) {
+UniqueBuffer Context::createVertexBuffer(const void* data, std::size_t size, bool dynamicDraw) {
```

Comment:
Can we please expose this as an enum similar to how the others work in `gl/types.hpp`?